scan/src/pages/ValidatorHomePage.re

```diff
@@ -205,7 +233,10 @@ let make = () =>
     let allValidatorCount =
       bondedValidatorCount + unbondedValidatorCount + unbondingValidatorCount;
 
-    let allBondedAmount = bondedValidatorCount;
+    //TODO: Replace with real value
+    let allBondedAmount = 400;
```

Comment:
We have this data already, let's not mock it?

Reply:
Fixedin in #1191